src/screens/SignUp/index.tsx

```diff
@@ -1,27 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
+import { useMutation } from 'react-query';
```

Comment:
I think that useMutations is for request has been cached and we need updated it, please check the use of the simple [fetcher](https://react-query.tanstack.com/overview#enough-talk-show-me-some-code-already)